Blockzilla/SettingsViewController.swift

```diff
@@ -350,18 +352,23 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
             searchCell.accessoryLabelText = accessoryLabel
```

Comment:
nit: can we rename accessoryLabel to a more descriptive name like accessoryLabelText since it's a string not a label? Doing the same for autocompleteLabel would be nice too--it's just a little confusing from a code readability standpoint.

Reply:
This isn't my code -> should I still change it since it's outside the scope of this PR?